### DISCUSSION TITLE
Integration along azimuthal direction in a pole figure

### DISCUSSION THREAD

**Discussion by mfuhr-mpg** [mtex-toolbox/mtex#1536] [Ask Anything] [answered]
Hi everyone,

I´m aware that this might be a very basic question but I would like to integrate a pole figure along the azimuthal direction in order to plot the integrated intensity as a function of the radius.

This is the code that I use in order to obtain my pole figure from an orientation map (EBSD):

`CS = crystalSymmetry('m-3m', [3.2 3.2 3.2], 'mineral', 'W', 'color', [0.53 0.81 0.98]);
l =  Miller(5,3,1,CS,'hkl');
ori=ebsd('W').orientations;
plotPDF(ori,l,'contourf','antipodal','projection','eangle');`

An example of a pole figure is attached.

Thanks in advance for any comments/suggestions!

![531_texture_150](https://user-images.githubusercontent.com/79193413/187469391-c67af573-decd-4abd-9006-6a578490fd20.png)

**Comment by ralfHielscher** [comment, marked as answer]
Hi,

you can do the following
```matlab
% compute a smooth pole figure
odf = calcODF(ori)
pf = odf.calcPDF(Miller(5,3,1,odf.CS,'hkl'));

% symmetrise the pole figure with respect to the z-axis
[pfSym,psi] = pf.symmetrise(vector3d.Z)

% plot the result
plot(psi)
```

Ralf.

**Reply by mfuhr-mpg** [reply]
Hi Ralf,

thanks for the suggestion!
The code works and looks like I wanted it to be, but I'm not quite sure about the dimensions. If I do `plot(psi)`, the x data lies between 0 and 180, so I assume that this is the azimuthal angle. I however expected to see a measure of the radius, so something in the range from 0 to 1. Is this just an error in understanding on my part, or is there a bug?

Best
Max

**Reply by ralfHielscher** [reply]
Hi Max,

a pole figure is a function on the sphere. Hence, the colatitude, i.e., angle from the north pole, is the appropriate measure. The radius depends on the spherical projections and hence is a bad choice.

Ralf.

**Reply by mfuhr-mpg** [reply]
Hi Ralf,
I see, thanks for the explanation!
Max
